conf/modules.config

```
@@ -124,17 +124,24 @@ process {
         publishDir = [
```

The error you are getting with "input file name collision" is because the files created by this process and the other `AGAT_` process have the same name, because they are named with `meta.id`. this is confusing, because you are *publishing* them with different names, however what is being passed to your `EXTRACT_` process is the original file (found in `work/../...`), and those have the exact same names. We can change how that file is named here:

```suggestion
        ext.prefix = { "${meta.id}_sqstats" }
```

---

Also note this is assuming removal of the full publishDir block for both of the nf-core AGAT processes